src/NuxeoClient/Operation.cs

```diff
@@ -390,24 +459,33 @@ public async Task<Entity> Execute()
                 data.Add("context", jsonCotext);
             }
 
-            bool isBlob = Input is Blob;
-            bool isBlobList = Input is BlobList;
-
-            if (Input != null && !isBlob && !isBlobList)
-            {
-                data.Add("input", (string)Input);
-            }
-
-            if (isBlob)
+            if (Input is Blob)
             {
                 return await client.RequestMultipart(Endpoint, data, (Blob)Input, HttpMethod.Post, headers);
             }
-            if (isBlobList)
+            if (Input is BlobList)
             {
                 return await client.RequestMultipart(Endpoint, data, (BlobList)Input, HttpMethod.Post, headers);
             }
             else
             {
+                ICollection<Document> docs = Input is Documents ? ((Documents)Input).Entries : Input as ICollection<Document>;
```

Comment:
Not sure if it the best approach, but I can understand that it may not be easy to use other approach (it will repeat more code and, necessarily, be verbose)